tests/test_utils.py

```diff
@@ -18,7 +18,12 @@
 
```

Comment:
I noticed that if I converted the data model by doing: 
```
schematic schema convert tests/data/example.model.csv --data_model_labels display_label
```
and then run the test, one test would fail: `tests/test_utils.py::TestCsvUtils::test_csv_to_schemaorg - assert '{\n    "@con...s.io/#0.1"\n}' == '{\n    "@con...s.io/#0.1"\n}'`

Is this expected?

Reply:
yes...bc that test is asserting that the test data model is the same as the saved one. I think you would need to make sure to not modify the example model in the test folder. 
so you could recreate it using the class_label option, then run the test again.